pennylane/ops/qubit.py

```diff
@@ -2142,9 +2168,6 @@ def decomposition(wires):
 
         return decomp_ops
 
-    def adjoint(self, do_queue=False):
-        return QFT(wires=self.wires, do_queue=do_queue).inv()
-
```

Comment:
For my understanding: this was removed because QFT and other template-like operations will now default to expand?  As you mentioned in #1200#r610612234 ?

Reply:
Yep!